migrate/20250212_add_provider_locations.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_table(:provider_name) do
+      column :name, :text, null: false, primary_key: true
+    end
+    create_table(:provider_location) do
+      column :id, :uuid, primary_key: true
+      column :display_name, String, null: false
+      column :internal_name, String, null: false
+      column :ui_name, String, null: false
+      column :visible, :boolean, null: false
+      foreign_key :provider_name, :provider_name, type: :text, null: false
+    end
+
+    run <<~SQL
```

Comment:
Sequel does not technically support multiple queries in a `run` command, though it does work on some adapters (including the one we are currently using).  You  can use two `run` calls, or you could switch to using `Sequel::Dataset#import` (see `20241215_authorization_refactor` for an example of that).